libs/vkd3d/state.c

```diff
@@ -4082,8 +4084,25 @@ VkPipeline d3d12_pipeline_state_create_pipeline_variant(struct d3d12_pipeline_st
     vp_desc.scissorCount = key && !key->dynamic_viewport ? max(key->viewport_count, 1) : 0;
     vp_desc.pScissors = NULL;
 
+    rtv_active_mask = key ? key->rtv_active_mask : graphics->rtv_active_mask;
```

Comment:
We should be able to dynamically render to NULL RTV with dynamic rendering, so not sure if we even need a key for rtv_active_mask here.

Reply:
See below, I'd rather land dynamic_rendering first and then do the optimization.